books/README.md

```diff
@@ -0,0 +1,99 @@
+# Books Dataset
+
+The books.json is a subset from the openlibrary [books datasets](https://openlibrary.org/developers/dumps)
+
```

Comment:
we would need to add the CC0 1.0 universal license here I think: https://openlibrary.org/help/faq/using#ownership

Reply:
@Haroenv To the best of my knowledge when it comes to CC0 1.0 universal license following rules apply.

- You may use the dataset for commercial purposes.
- No need to cite or reference the license.
- Attribution is optional, not required.

Reply:
Thanks for digging in on the licensing, Ankur. Based on your research I agree with you.